src/java/fr/paris/lutece/portal/web/dashboard/DashboardJspBean.java

```diff
@@ -150,7 +150,7 @@ public String doMoveDashboard( HttpServletRequest request ) throws AccessDeniedE
 
             if ( AppLogService.isDebugEnabled( ) )
             {
```

Comment:
It is unnecessary to check if logging is enabled because it is already managed with the debug level.